modules/rosa-architecture.adoc

```diff
@@ -1,21 +1,29 @@
 // Module included in the following assemblies:
 //
 // * rosa_architecture/rosa_architecture_sub/rosa-architecture-models.adoc
-[id="rosa-architecture_{context}"]
-= ROSA architecture on public and private networks
 
-You can install ROSA using either a public or private network. Configure a private cluster and private network connection during or after the cluster creation process.
-Red Hat manages the cluster with limited access through a public network. For more information, see xref:../../rosa_architecture/rosa_policy_service_definition/rosa-service-definition.adoc#rosa-service-definition[ROSA service definition].
+[id="rosa-classic-architecture_{context}"]
+= ROSA Classic architecture
 
-.ROSA Classic deployed on public and private networks
-image::156_OpenShift_ROSA_Arch_0621_private_public_classic.png[ROSA deployed on public and private networks]
+In ROSA Classic, both the control plane and the worker nodes are deployed on the customer's VPC subnets.
+
+[id="rosa-classic-architecture-networks_{context}"]
+== ROSA Classic architecture on public and private networks
+
+With ROSA Classic, you can create your clusters on public or private networks.
 
-If you are using {hcp-title-first}, you can create your clusters on public and private networks as well. The following images depict the architecture of both public and private networks.
+Customers can customize access patterns for their API server endpoint and Red Hat SRE management in the following ways:
```

Comment:
In the preview, the bullet points are not wrapped/formatted. can you please check?